caesars-palace/blocks/slider-card/slider-card.js

```diff
@@ -51,17 +84,20 @@ export default function decorate(block) {
 
   const mediaWidthChangeHandler = (event) => {
     if (event.matches) {
-      const shortDesriptionTabletDivs = document.querySelectorAll('.short-description');
+      const shortDesriptionTabletDivs = block.querySelectorAll('.short-description');
       shortDesriptionTabletDivs.forEach((div) => {
         div.classList.add('hide');
         div.classList.remove('show');
       });
 
-      const longDescriptionDivs = document.querySelectorAll('.long-description');
+      const longDescriptionDivs = block.querySelectorAll('.long-description');
       longDescriptionDivs.forEach((div) => {
         div.classList.add('show');
         div.classList.remove('hide');
       });
+
+      const sliderCard = block.closest('.slider-card');
+      sliderCard.style = 'width: 100%;';
```

Comment:
Can this not be done using CSS only?

Reply:
When we change the orientation from Desktop to Mobile -we see the desktop version, unless the width is modified here.